src/features/projects/components/maps/Explore.tsx

```diff
@@ -0,0 +1,443 @@
+      !!decodeConfig &&
+      getParams(decodeConfig, { ...timelineConfig, ...lSettings.decodeParams });
+    const timelineParams = !!timelineConfig && {
+      ...timelineConfig,
+      ...getParams(decodeConfig, lSettings.decodeParams),
+    };
+
+    return {
+      id,
+      slug: id,
+      dataset: id,
+      layers: [
+        {
+          active: true,
+          ...l,
+          ...lSettings,
+          decodeParams,
+          timelineParams,
+        },
+      ],
+      ...lSettings,
+    };
+  });
+
+  const onChangeLayerDate = (dates: any, layer: any) => {
+    const { id, decodeConfig } = layer;
+
+    setLayersSettings({
+      ...layersSettings,
+      [id]: {
+        ...layersSettings[id],
+        ...(decodeConfig && {
+          decodeParams: {
+            startDate: dates[0],
+            endDate: dates[1],
+            trimEndDate: dates[2],
+          },
+        }),
+        ...(!decodeConfig && {
+          params: {
+            startDate: dates[0],
+            endDate: dates[1],
+          },
+        }),
+      },
+    });
+  };
+
+  const handleExploreProjectsChange = (event: any) => {
+    setExploreProjects(event.target.checked);
+    setShowProjects(event.target.checked);
+    if (!event.target.checked) {
+      const newViewport = {
+        ...viewport,
+        latitude: 36.96,
+        longitude: 0,
+        zoom: 1.4,
+        transitionDuration: 1200,
+        transitionInterpolator: new FlyToInterpolator(),
+        transitionEasing: d3.easeCubic,
+      };
+      setViewPort(newViewport);
+    } else {
+      // const newMapState = {
+      //   ...mapState,
+      //   mapStyle: 'mapbox://styles/sagararl/ckdfyrsw80y3a1il9eqpecoc7',
+      // };
+      const newViewport = {
+        ...viewport,
+        latitude: defaultMapCenter[0],
+        longitude: defaultMapCenter[1],
+        zoom: 1.4,
+        transitionDuration: 1200,
+        transitionInterpolator: new FlyToInterpolator(),
+        transitionEasing: d3.easeCubic,
+      };
+      // setMapState(newMapState);
+      setViewPort(newViewport);
+      router.push('/', undefined, {
+        shallow: true,
+      });
+    }
+  };
+
+  React.useEffect(() => {
+    document.addEventListener('mousedown', (event) => {
+      if (exploreExpanded) {
+        if (
+          exploreContainerRef &&
+          exploreContainerRef.current &&
+          !exploreContainerRef.current.contains(event.target)
+        ) {
+          setExploreExpanded(false);
+        }
+      }
+    });
+  });
+
+  React.useEffect(() => {
+    if (exploreExpanded) {
+      setMapState({ ...mapState, dragPan: false });
+    } else {
+      setMapState({ ...mapState, dragPan: true });
+    }
+  }, [exploreExpanded]);
+
+  return (
+    <>
+      {exploreForests ? (
+        <Source
+          id="forests"
+          type="raster"
+          tiles={[
+            'https://tiles.arcgis.com/tiles/lKUTwQ0dhJzktt4g/arcgis/rest/services/Forest_Denisty_V2/MapServer/tile/{z}/{y}/{x}',
```

Comment:
Can we be sure this URL does not change. Should be extract it into an environment variable?

Reply:
This is hosted in our esri account so not unless someone decides to delete it. and if it changes it won't crash the app.